sormas-backend/src/main/java/de/symeda/sormas/backend/region/GeoShapeHelper.java

```diff
@@ -134,21 +138,21 @@ public static MultiPolygon getPolygon(SimpleFeature feature, MathTransform trans
 	 * @return
 	 *         The name of the feature, null if no name could be found.
 	 */
-	public static String sniffShapeName(SimpleFeature feature, List<String> list) {
-		String shapeName = null;
+	public static String sniffShapeAttribute(SimpleFeature feature, List<String> list) {
```

Comment:
Rename list to attributeNames.
I think the old comment also needs to be adjusted. It's returning the value of the found feature attribute, not the attribute name.